### DISCUSSION TITLE
[@types/jwplayer] missing functions

### DISCUSSION THREAD

**Discussion by pkmnct** [DefinitelyTyped/DefinitelyTyped#56870] [Issues with a @types package]
Hello!
It seems the type file for [@types/jwplayer] is missing a handful of newer functions. 
https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/jwplayer

The docs have the available functions:
https://developer.jwplayer.com/jwplayer/docs/jw8-javascript-api-reference

I noticed that, at a minimum, `jwplayer().getFloating()` and `jwplayer().setFloating()` are missing, but I anticipate there are additional functions.

[a comment by an automated account is not shown]

**Comment by djcsdy** [comment]
It's not unusual for the type definitions to lag behind the library by some amount. Please by all means create a PR that adds definitions for some or all of the missing functions. Any contribution that improves the type definitions is welcome.

**Comment by martin-badin** [comment]
PR is ready https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63737. Please review
